migration/handler.go

```diff
@@ -0,0 +1,52 @@
+package migration
+
+import (
+	"github.com/iov-one/weave"
+	"github.com/iov-one/weave/errors"
+	"github.com/iov-one/weave/x"
+)
+
+// RegisterRoutes registers handlers for feedlist message processing.
+func RegisterRoutes(r weave.Registry, auth x.Authenticator) {
+	bucket := NewSchemaBucket()
+	r.Handle(pathUpgradeSchemaMsg, &upgradeSchemaHandler{bucket: bucket})
+}
+
+type upgradeSchemaHandler struct {
+	bucket *SchemaBucket
+}
+
+func (h *upgradeSchemaHandler) Check(ctx weave.Context, db weave.KVStore, tx weave.Tx) (*weave.CheckResult, error) {
+	if _, err := h.validate(ctx, db, tx); err != nil {
+		return nil, err
+	}
+	return &weave.CheckResult{}, nil
+}
+
+func (h *upgradeSchemaHandler) Deliver(ctx weave.Context, db weave.KVStore, tx weave.Tx) (*weave.DeliverResult, error) {
+	msg, err := h.validate(ctx, db, tx)
+	if err != nil {
+		return nil, err
+	}
+
+	ver, err := h.bucket.CurrentSchema(db, msg.Pkg)
+	if err != nil && !errors.ErrNotFound.Is(err) {
```

Comment:
Nice, so this same message will upgrade form 0 to 1 (ErrNotFound) to enable it in the first place. I like the symetry